conf/layer.conf

```
@@ -52,6 +52,8 @@ BBFILES_DYNAMIC += " \
     \
     meta-arm:${LAYERDIR}/dynamic-layers/meta-arm/*/*/*.bbappend \
     \
+    meta-arm-toolchain:${LAYERDIR}/dynamic-layers/meta-arm-toolchain/*/*/*.bb \
```

I believe the layer collection uses a different name. Have you tested that?

```suggestion
    arm-toolchain:${LAYERDIR}/dynamic-layers/arm-toolchain/*/*/*.bb \
```

My understanding is that you should be able to build the recipe just with the `meta-arm-toolchain` layer included without the `meta-arm` layer being listed in the `bblayers.conf`.

---

Thanks @otavio. I pushed these changes now, I tested with 

```
MACHINE=imx95-19x19-verdin bitbake imx-oei
```

from meta-imx-bsp, using this patch inside meta-freescale and the build passes.

---

I believe that the file is in the wrong place, my understanding is that it should be located here:

```
├── arm-toolchain
│   └── recipes-bsp
│       └── imx-oei
│           └── imx-oei_1.0.0.bb
```

This is because of the collection of the layer. See:

https://git.yoctoproject.org/meta-arm/tree/meta-arm-toolchain/conf/layer.conf#n7

I suspect you have the previous file in some location that could be found. Otherwise, the build would fail.

Could you double check that? In worst case, we could jump in a short call.

---

Hi @otavio, sorry for the delay. Now I moved to the correct folder and did a quick build again, and it looks like it is working (I did a "cleanall" this time). Can you please take a look? Thank you!